pkg/build/api/v1/types.go

```diff
@@ -539,6 +539,10 @@ type BuildConfigStatus struct {
 type WebHookTrigger struct {
 	// Secret used to validate requests.
 	Secret string `json:"secret,omitempty"`
+
+	// AllowEnv determines whether the webhook can set environment variables; can only
```

Comment:
ok ... looks like I should go ahead and change the above `Secret` to `secret` as well, even perhaps review the file in general.